### PR TITLE
[FIX] Basket seed time and Grub Shop Prices

### PR DESCRIPTION
**Pull request by mGranTY**
# Description

Fix Basket Seed time being inconsistent with actual crop
##
Also fix grub shop prices for cakes not showing the boost
Before: 
image
After:
image
 

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Check your basket seed time and the planted crop time

# Checklist:

- [X] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [X] Screenshot if it includes any UI changes
- [X] I have read the contributing guidelines and agree to the T&Cs
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/island/hud/components/inventory/Basket.tsx [hunk 1]
```diff
@@ -30,10 +33,14 @@ import { BEANS } from "features/game/types/beans";
...
 
+interface Props {
+  state: GameState;
+}
+
 const isSeed = (selectedItem: InventoryItemName) =>
   selectedItem in CROP_SEEDS();
 
-export const Basket: React.FC = () => {
+export const Basket: React.FC<Props> = ({ state }: Props) => {
```
Comment (spencerdezartsmith) [inline src/features/island/hud/components/inventory/Basket.tsx:43]: Looks like this component already has access to the state without needing to pass this is as a prop.


You should just be able to pull the necessary values off the `gameState.context.state`
```
const { inventory, bumpkin, collectibles } = gameState.context.state;
```

File: src/features/island/hud/components/inventory/InventoryItems.tsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ export const InventoryItems: React.FC<Props> = ({ state, onClose }) => {
         />
       </div>
 
-      {currentTab === "basket" && <Basket />}
+      {currentTab === "basket" && <Basket state={state} />}
```
Comment (spencerdezartsmith) [inline src/features/island/hud/components/inventory/InventoryItems.tsx:72]: No need for `state` prop based on above comment.

**Review by spencerdezartsmith — CHANGES_REQUESTED** [review]
Thanks for jumping on this. Just some comments.

File: src/features/island/hud/components/inventory/Basket.tsx (hunk 1 above)
Comment (mGranTY) [reply src/features/island/hud/components/inventory/Basket.tsx:43]: how did i miss that? 😥

**Review by spencerdezartsmith — APPROVED** [review]
🚀

**Review by Rofrtd — APPROVED** [review]
LGTM

**Pull request merged**